src/index.tsx

```diff
@@ -23,5 +21,5 @@ root.render(
         </Provider>
       </AuthProvider>
     </BrowserRouter>
-  </React.StrictMode>
+  </React.StrictMode>,
```

Comment:
Is there any way for us to remove the comma that the formatter implements after JSX elements?

Reply:
I read up on why this is the default in the first place and it looks like it's for consistency, cleaner code reviews (the diffs apparently look better), and easily moving stuff around. For example, if you want to move items between a list around or parameters for functions around having the trailing comma makes it so that you can cut and paste the new line without having to add a new comma

It turns out this comma isn't on trailing JSX; it's because the JSX is in a function (`root.render()`). We are able to set `trailingComma` to be `es5` in the `.prettierrc` file to make it so that we have trailing commas on arrays but not function parameters. I don't really mind having the commas, but if it really bothers us I'm not against setting it to `es5`. But now knowing all that stuff do you think you want to change it?